app/src/main/java/org/oppia/android/app/preview/TopicPreviewActivity.kt

```diff
@@ -0,0 +1,52 @@
+package org.oppia.android.app.preview
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** TopicPreviewFragment TAG. */
+const val TOPIC_PREVIEW_FRAGMENT_TAG = "TopicPreviewFragment"
+private const val TOPIC_PREVIEW_ACTIVITY_TOPIC_ID_ARGUMENT_KEY = "TopicPreviewActivity.topic_id"
+private const val TOPIC_PREVIEW_ACTIVITY_PROFILE_ID_ARGUMENT_KEY = "TopicPreviewActivity.profile_id"
+
+/** Activity display preview of topic when topic is not downloaded. */
+class TopicPreviewActivity : InjectableAppCompatActivity() {
+
+  companion object {
+    /**
+     * This function creates the intent of [TopicPreviewActivity].
+     *
+     * @param [internalProfileId]: Id of the Profile.
```

Comment:
```suggestion
     * @param internalProfileId Id of the profile
```

Note:
- The brackets aren't needed here since it's already expected that the name will match a parameter in the parameter list
- The colon should not be here
- Most words should not be capitalized unless they are proper nouns ('profile' is not a proper noun in this context)
- At-clauses don't end in punctuation unless they are multiple sentences

Ditto elsewhere in this PR & future PRs.